### PR TITLE
SOC-3879 title is now required

### PR DESCRIPTION
**Pull request by slayful**
https://wikia-inc.atlassian.net/browse/SOC-3879

### CODE REVIEW

File: front/main/app/components/discussion-editor.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import Ember from 'ember';
-import {track} from '../utils/discussion-tracker';
-import DiscussionEditorConfiguration from '../mixins/discussion-editor-configuration';
-import DiscussionEditorOverlayMessage from '../mixins/discussion-editor-overlay-message';
+import Ember from "ember";
```
Comment (bkoval) [inline front/main/app/components/discussion-editor.js]: Why a change to double quotes?

File: front/main/app/components/discussion-editor.js [hunk 2]
```diff
@@ -56,12 +56,13 @@ export default Ember.Component.extend(
...
 		}),
 
-		submitDisabled: Ember.computed('content', 'currentUser.isAuthenticated', 'showOverlayMessage',
+		submitDisabled: Ember.computed('content', 'title', 'currentUser.isAuthenticated', 'showOverlayMessage',
 			'isReply', 'category', 'isGuidelinesEditor', function () {
 				return this.getWithDefault('content.length', 0) === 0 ||
 					this.get('currentUser.isAuthenticated') === false ||
 					this.get('showOverlayMessage') ||
-					(!this.get('isReply') && !this.get('category.id') && !this.get('isGuidelinesEditor'));
+					(!this.get('isReply') && !this.get('category.id') && !this.get('isGuidelinesEditor')) ||
```
Comment (bkoval) [inline front/main/app/components/discussion-editor.js]: How about we split this logic to be a bit more readable? Either splitting the if conditions or abstracting checks to separate methods would help.

File: front/main/app/templates/components/discussion-standalone-editor.hbs [hunk 3]
```diff
@@ -2,7 +2,8 @@
 	<a href="#" class="editor-close" {{action 'close'}}>
 		{{svg 'close' class="fill-theme-color" viewBox='0 0 18 18'}}
 	</a>
-	<button class="active-element-theme-color discussion-standalone-editor-save-button" type="submit" {{action 'submit'}} disabled={{submitDisabled}}>{{i18n submitMessageKey ns='discussion'}}</button>
+	<button class="active-element-theme-color discussion-standalone-editor-save-button" type="submit"
+		{{action 'submit'}} disabled={{submitDisabled}}>{{i18n submitMessageKey ns='discussion'}}</button>
```
Comment (bkoval) [inline front/main/app/templates/components/discussion-standalone-editor.hbs]: How about you move the </button> to the new line as well?

File: front/main/app/components/discussion-editor.js (hunk 1 above)
Comment (slayful) [inline front/main/app/components/discussion-editor.js]: Due to our coding style. Linter will complain otherwise.

**Pull request merged**